packages/frontend-core/src/components/SplitPage.svelte

```diff
@@ -25,6 +25,7 @@
     overflow-y: auto;
   }
   .right {
+    overflow: hidden;
```

Comment:
Was having trouble with elements not respecting 100% width here and overflowing until I added this. I wonder if it's because this is the child of a grid parent?

Reply:
Yea that's fine to add it. I have added position relative in a separate commit which is what is conflicting here. That might actually also solve this problem, but it's fine to have both.